models/core/transfers.sql

```diff
@@ -0,0 +1,40 @@
+{{ config(
+  materialized='incremental', 
+  unique_key= "CONCAT_WS('-', 'block_id', 'tx_hash', coalesce('from_address',''), coalesce('to_address',''), coalesce('contract_address',''), coalesce('raw_amount',-1))",
+  incremental_strategy = 'delete+insert',
+  cluster_by = ['block_timestamp'],
+  tags=['core']
+) }}
+
+with logs as (
+  select 
+    q.block_id, 
+    q.block_timestamp, 
+    q.tx_hash,
+    q.evm_contract_address, 
+    q.event_name, 
+    q.event_inputs
+  from 
+    {{ ref('logs') }} q
+),
+transfers as (
+  select
+    block_id,
+    tx_hash,
+    block_timestamp,
+    evm_contract_address::string as contract_address,
```

Comment:
There is no ::string in the logs table:

value:address as evm_contract_address,

This will leave the contract_address with this format " " while the rest are in ' '. Is this intended?

Reply:
oh i think that'd be the issue for the upstream `log` table, which should have added the type there.
cc: @forgxyz

Reply:
@ndhung1104 since log is now updated, you can remove parsing data type in this query logic now :)